tests/runtests.py

```diff
@@ -82,7 +82,7 @@ def test_cmd_dereference(self):
         self.assertTrue(len(res.splitlines()) > 2)
         self.assertIn("$rsp", res)
 
-        res = gdb_start_silent_cmd("dereference 0")
+        res = gdb_start_silent_cmd("dereference 0x0")
```

Comment:
Why?

Reply:
If it is 0, then it will be parsed as the number of lines instead of the address. To make it sure that it is parsed as an address to start dereferencing from, it should be 0x0.

Reply:
Ok, that's _probably_ fine (let's ask @hugsy ) but in general we prefer to be explicit: I'd rather the args have a defined order than have someone try to dereference an address and have it be interpreted as a number of lines. Using a non-hex address might be rare, but when it happens, it's probably in the thousands and will cause issues. imagine a vuln that leaks an address in decimal. If I copy and paste that into deref, with this change I'd be in trouble.

Reply:
Yes, the only case it can cause a problem is the address might be given in decimal. However, it is easy to convert addresses to hexadecimal and also you probably would be leaking an address in hexadecimal. I always assume addresses are in hexadecimal. For example,if I press Ctrl+G in Ollydbg and type 1000, I expect it to go to 0x1000 instead of 1000. In general, I think this change, makes it easier and more flexible to use.